lib/lndclient/LndClient.ts

```diff
@@ -42,6 +42,7 @@ class LndClient extends BaseClient{
     } else {
       const lndCert: Buffer = fs.readFileSync(certpath);
       const credentials: ChannelCredentials = grpc.credentials.createSsl(lndCert);
+      const rpcprotopath = path.join(__dirname, '..', '..', '..', 'lndrpc.proto');
```

Comment:
1) can be written as `path.join(__dirname, '../../../lndrpc.proto')`. If you knew it and preferred the above, you can leave 
1) any reason for not putting `lndrpc.proto` inside of `lndclient` folder? (considering `lndrpc_pb.d.ts` is already there)

Reply:
Yeah I knew it could be either way, seems like both ways are equally good and effective so I will just leave as is.

As for putting `lndrpc.proto` inside of `lndclient`, I had thought of it but then decided against it. It's not part of our code and it doesn't get compiled, so it seemed like it would be out place in `lib` (`lndrpc_pb.d.ts` does get compiled and is part of our source code). Also, I thought it might cause a problem because the file will actually run within the `dist` folder and I wasn't sure if `.proto` files would get moved over from `lib` to `dist`, maybe it would but I didn't bother testing it.